p2p_wallet/Scenes/Main/SendToken/Subscenes/ChooseRecipientAndNetwork/SelectAddress/SendToken.ChooseRecipientAndNetwork.SelectAddress.RootView.swift

```diff
@@ -110,6 +125,12 @@ extension SendToken.ChooseRecipientAndNetwork.SelectAddress {
                 if viewModel.relayMethod == .relay {
                     feeView
                 }
+                warningView.setup {
```

Comment:
the .setup function only makes sense when using declarative syntax. Now just use viewModel.warning.drive(warningView.rx.isHidden)